src/NewRemotingUnitTest/MessageHandlerTests.cs

```diff
@@ -60,15 +83,44 @@ public void SerializeException()
 			MemoryStream ms = new MemoryStream();
 			BinaryWriter bw = new BinaryWriter(ms);
 			BinaryReader br = new BinaryReader(ms);
-			MessageHandler.EncodeException(x, bw);
+			Handler.EncodeException(x, bw);
 			long encodingLength = ms.Position;
 			ms.Position = 0;
-			var decoded = MessageHandler.DecodeException(br, "Dummy");
+			var decoded = MessageHandler.DecodeException(br, "Dummy", Handler);
 			long decodingLength = ms.Position;
 			Assert.IsNotNull(decoded);
 			Assert.True(decoded is RemotingException);
 			Assert.Null(decoded.InnerException);
 			Assert.AreEqual(encodingLength, decodingLength);
 		}
+
+		[Test]
+		public void SerializeExceptionWithProperties()
+		{
+			OperationCanceledException c = new OperationCanceledException();
+			c.Source = "some source";
+
+			var ldue = new LidarUnitException("Exception during sky lidar unit sensor communication",
```

Comment:
This is public, so maybe change these strings.

Reply:
done